lib/pcb/create-svg-objects-from-pcb-rats-nest/get-source-port-for-pcb.ts

```diff
@@ -0,0 +1,9 @@
+import type { AnyCircuitElement } from "circuit-json"
+import { su } from "@tscircuit/soup-util"
+
+export const getSourcePortForPcbPort = (
+  pcbPortId: string,
+  soup: AnyCircuitElement[],
+) => {
+  return su(soup).source_port.getWhere({ pcb_port_id: pcbPortId })
```

Comment:
so when you have a function that is so simple that it only has a line or two, you actually increase the complexity of the codebase by increasing the "go-to-definition-depth" (or "complexity depth") of the code, so you should remove this function and call `su` directly from the call site